fava/plugins/link_statements.py

```diff
@@ -20,6 +22,8 @@ def link_statements(entries, options_map):
     errors = []
 
     if 'documents' not in options_map or len(options_map['documents']) == 0:
+        errors.append(LinkStatementError(
+            'link_statements requires "documents" option to be set'))
```

Comment:
The plugin should also work if there's no document folder (and all documents are given as entries in the file), so this shouldn't be an error.